### PR TITLE
TV Shows API Webapp

### PR DESCRIPTION
**Pull request by TedLivist**
#### In this project, we developed a TV Show app where users can like and comment on any show.
Below are some of the steps we took to complete the project. We:
- Selected an API. In this project, we used the [TV Maze Shows API](https://api.tvmaze.com/shows) as the external API
- Set up the project with `Webpack` and installed the `Jest` testing library.
- Added the `footer` and the `header`
- Fetched a list of TV Shows from the TV Maze API and displayed them on the Homepage, with associated `❤️` icon and `Comment` button
- Built the Comments modal popup for each TV Show with associated details
- Displayed the number of likes for each TV Show and allow users to add new like using the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270)
- Displayed the comments for each TV Show and allow users to add new comments using the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270)
- Added the number of TV Shows fetched from the TV Maze API to the homepage
- Displayed the number of comments for each TV Show on the Comments modal
- Added the tests for the TV Shows counter and the Comments counter

#### Here is the link to our Zoom presentation: [Zoom Link](https://cutt.ly/SWGx8hv)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+- HTML, CSS and JavaScript
+- Webpack
+- Jest
+
+## Live Demo
+Visit the live demo [here](https://bit.ly/3tmGmS2)
+
+## Getting Started
+
```
Comment (carlos-ssh) [inline README.md:19]: _- Kindly work on these changes:_
**Project documentation - video presentation**

1. The maximum length of the video is 5 minutes.
2. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
3. All the members in the video should appear in the video and talk about the project.
4. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TedLivist) [reply README.md:19]: Hello @carlos-ssh, we don't quite under what changes are being requested.

The link to the video presentation is on the Pull Request description.
Maybe you could be more explicit in explaining what is wrong with the video.
- Our video length is less than 5 minutes.
- The beginning of the video describes the scope of the project.
- All members of the group appeared in the video. Me and @julie-ify.
- We reviewed a cross-section of the source code we implemented.

Comment (carlos-ssh) [reply README.md:19]: Thanks for comment. I tried many times to load the video and i had an error loading it on google.

Comment (TedLivist) [reply README.md:19]: Okay. Maybe it has to do with your PC? The link works just fine on my computer. I just clicked it now and it loaded the video

**Review by carlos-ssh — APPROVED** [review]
Hi @TedLivist , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**